LibSaberPatch/Apk.cs

```diff
@@ -60,6 +60,39 @@ public class Apk : IDisposable
             }
         }
 
+        /// <summary>
+        /// Deletes the given file from the APK
+        /// </summary>
+        /// <param name="filePath">The file to delete in the APK</param>
+        public void RemoveFileAt(string filePath)
+        {
+            ZipArchiveEntry entry = archive.GetEntry(filePath);
+            if (entry != null) entry.Delete();
+        }
+
+        public void WriteSplits(string basePath)
```

Comment:
There was an exception after using uber-signer.jar and then attempting to run adb install which looked like this:
```Performing Streamed Install
adb: failed to install base.apk: Failure [INSTALL_PARSE_FAILED_NO_CERTIFICATES: Failed to collect certificates from /data/app/vmdl1854736680.tmp/base.apk: /data/app/vmdl1854736680.tmp/base.apk: File assets/bin/Data/sharedassets17.assets.split4 in manifest does not exist]```

This could have been due to many things, but I assumed that the sharedassets17 file not being split caused this to happen. It _might_ not be necessary, but I felt like it was better to have the method than not. The reason I read from the file is because I have it set up to immediately write using the existing path, in an attempt to remain similar to the format of `JoinedContents`. I fixed some of this in commit: https://github.com/trishume/QuestSaberPatch/pull/2/commits/00f2cad57d15af6d9334ed1f4e36acc481e5ea0d